webapp/src/js/controllers/reports.js

```diff
@@ -314,6 +315,10 @@ angular
           })
           .catch(function(err) {
             $log.error('Error verifying message', err);
+          })
+          .finally(() => {
+            $scope.selected[0].verified = valid;
```

Comment:
Sorry, I think this should be `$scope.selected[0].doc.verified` (notice the `.doc` in there). You can test this by slowing down or even temporarily disabling the Changes service and making sure the status is set correctly.